app/views/events/_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_for event do |form| %>
+<%= form_for [organization, event] do |form| %>
```

Comment:
We can figure out the event's organization from the event alone here, too. No need to pass in organization separately where this partial is used.